pandas/_libs/tslibs/np_datetime.pyx

```diff
@@ -304,6 +304,7 @@ cpdef ndarray astype_overflowsafe(
     cnp.dtype dtype,
     bint copy=True,
     bint round_ok=True,
+    bint coerce=False,
```

Comment:
perhaps `is_coerce`, for consistency with `tslib`?

Reply:
sure